notebooks/Linear Regression.ipynb

```diff
@@ -0,0 +1,1288 @@
u4bKlABRXEDRBFw12HEUYef\njo4LsiOI7CBwyyIgaxdauidpkt8faUrb3GxN0jbt+3ken8g95557kib3nu973uXoEebPf4dHHnki\nqNcU93hBEOo1NpuN2bPfoG/f7uzfv1ezj16vF8EuCHXAunW/Mnhwf0aNGuE2jq958+YeBXtlnAYX\nEeyCIAiCN0wWK4czCzicVVgrlUZKSkqYPftN+vXryXvvzXMR7ABXXnk111wzNOjXlp12QRDqLSdO\nHOfBB0ezcuUyAO6/fyTff78Eg8FQxzNrmCiK0gJIB64FmgMngSXAJFVV9/tw/irgIjfNY1VVdU3z\nL4QlpaWlTJ8+lXffnYPdbufQoQO8/fYsHnroMe8nC4IgCEIAWG02/vXLHn7dfoxSsw2AaEMEA7q3\n4PbLOhOhD+6+tM1m48svFzF9+lSOHDms2adXr96kp79I//7ulkGBIaJdEIR6yS+//MT48WPIzs6u\nOLZt2xamTUtnypSX6mxeDZVywb4eaAP8DHwOKMBw4GpFUS5QVXWPl2F6AGr5udXZEMTpCnXI1q2b\nGTduNKq6u8rxV155kcsvv4pu3c6qo5kJgiAIjYFFS/eydOORKsdKzVZ+2XgEnU7H8CFdgnatlSuX\n88ILE9m+fatme9u27Xj22UkMG3YT+iAbCyojol0QhHqFyWRi2rR05s1726UtOTk5ZBZMgXQcgv1x\nVVXfcB5UFOVO4FPgdeB6dycritIeSADeV1U1PZQTFeoGi8XCzJmv8eabr2q6BN599yjatWsf9Oua\nLFaJcxcEQRAAxzNhk5rptn1zRhY3DeoY8PPCZDJxzz1/1ywtDJCYmMSjjz7JqFH3YzSGvjSpiHZB\nEOoN+/btYfToUWzbtsWlrX//i5g79z1atmxVBzNrFNwIZAEzKx9UVfUzRVHSgSsVRdGrqmpzc36P\n8tdtoZuiUFfs3LmT4cPvZOvWzS5trVu3Ydasdxg4cFBQr2m12Vi0dC+bM7I4mW8iJcFI7y5p3HZp\np6C7PgqCIAjhwalCEycLzG7bTxaYOFVoCjghrdFoJDo6xuW4wWBg5Mj7efTRJ0hOTgnoGv4gTz1B\nEOoFixYt5LLLLnYR7BERETzzzPN89dW3IthDhKIoEcBLQLobUW4CDECUh2FEtDdAbDYbc+fOoU+f\nPpqC/Y477mT58l+DLtjB4f64ZMNhcvJN2IGcfBNLNhxm0VLthJSCIAhCwycxzkhKvPvcRinxRhLj\ngrPz/fzzk4mMPL3HPWzY31i9+nemTHmpVgU7yE67IAh1TGFhAU899RhffrnIpa116zbMnfs+559/\nQR3MrPGgqqoVmKXVpihKV6ArsE9VVZOHYXoAduAiRVHewxEPnwt8CUxWVfVUcGcthJqDBw/w0ENj\nWbt2jUtbWlozXn99NldddU1Irm2yWNmckaXZtjkjOyiuj4IgCEL9wpdwKGNUBH2UZizZoJ0QrneX\nNL+eD2VlZRQU5GuK8A4dOnHPPfeyffs20tOn0adPX5/HDTYi2gVBqDO2bNnE/fffw4EDf7q0XXfd\nDbzxxmySkpLrYGYCgKIoemAODq+s+V669wB0wBQcQn0lMAh4GLhMUZQBqqrmh3C6EvscJOx2O599\n9jGTJj1LUVGhS/vQocOYMeNNUlNTQzaHU4UmTuZr24hyC0qD4vooCIEQaLUNQRBO42841G2XdsJm\nt/Pr9uOUmh2l3pzZ42+7tJNP17Tb7fz88w9MmTKJzp0VPvzwM81+kyZNxWAwoNPpav4Gg4CIdkEQ\n6oRdu3ZyzTVDXBJaRUdHM23aK9x11911foNszCiKogPmAZfhyPw+00NfPZAHbAGuU1X1SKXj7wCj\ncSxuPdYDS06OJTLSf7Fttdr44Ns/WLfjGFl5JaQlxXDBOS0ZOfRsIiJcH/ZpafF+X6MxYbfbWbNm\nuYtgT05O5u233+b2228P+W8zPjGGtOQYMnNLXNqaJsXQsX0q0YbwWMLI981/6vtnFqRqG4IglOMM\nh3LiDIcCNDPBR+j13Hm5wi2DO5GVWww6HWlJMT4b7Ldu3Ux6+vOsWbMKgIwMld9+W6fp2ekpyVxt\nbhaExxNPEIQGR9eu3bjmmqF8881/Ko5163YW8+Z9SNeu3epwZoKiKJHAAuBuYD9wg6qqbrO+lMfB\nuzzpVFW1KYryBHAXcAdeRHtubnGN5rtwSUaVh31mbgnfrNpPcYnZ5WGflhZPVlZBja7TmJg27TVW\nrlxFdrbDRf2qq65ixoxZtGjRkuxs1933UNCjY6qm+2OPjqkUnCohHP6K8n3zH63PrB6K+HQCqLYh\nCMJpPGWD36S6zwTvFMxpybE+C+ZDhw7y0ktT+Prrf7u0vfDC83z33c8+GaXrIlGqJKITBKFO0Ol0\nvP76LNq0aQs4ykX98MMyEex1jKIoscD/4RDse4BLVFU9WtPxVFUtBDKAFoqiRAdlkpXwFvtssliD\nfclGQdOmTXn99dnExjbhtddm8f3339OiRctancNtl3ZiSN/WpCZEo9dBakI0Q/q29tn1URBCiNtq\nG8A+yqtt1MXEBCHc8JQN3pkJvjJWm42FSzJ4fsE6Jsxbx/ML1rFwSQZWm7viNpCXl0t6+vP073+u\npmDX6XR06tSZkhJX7y4t6iJRquy0C4JQZyQmJvHuu+9z4sQJrrtONiXqGkVRkoH/AecDm4GrVNVD\nMdTT5yUBZwHZqqpmaHSJAWyAJYjTBST2OVC2b99K9+49NduuvvpaNmzYTtOmTeskVCVCr2f4kC7c\nNKij5CoQ6g2Vqm1YfKi24Sl5pyAIQIwxEr0ObHbXNr3O0V4Zf1zpzWYzM2fOZMqUKeTm5mpef/Dg\nS5k0aSrnnNPdp/nWVaJUsQIKghBS/vxzP2+95TYcmvPOO18Eez2gfBf8vzgE+wpgsC+CvZw+wBrg\nNY1xWwIdgM3
```

Comment:
I don't think this comment is accurate anymore -- aren't you plotting a distribution of test scores, rather than the means of the test and train scores?